content/kubermatic/master/tutorials_howtos/OPA_integration/Default Constraints/_index.en.md

```diff
@@ -0,0 +1,138 @@
++++
+title = "Default Open Policy Agent (OPA) Constraints"
+date = 2021-09-08T12:07:15+02:00
+weight = 20
+
++++
+
+### Default Constraints
+
+Default Constraints allow admins to conveniently apply policies to all OPA enabled clusters.
+This would allow admins an easier way to make sure all user clusters are following some policies (for example security), instead of the current way in which Constraints need to be created for each cluster separately.
+Kubermatic operator/admin creates a Constraint in the admin panel, it gets propagated to seed clusters and user clusters with OPA-integration.
+
+More Info about the KKP OPA integration can be read from these docs and tutorials,
+[OPA Integration](https://docs.kubermatic.com/kubermatic/master/tutorials_howtos/opa_integration/)
+
+### Managing Default Constraints
+
+Default Constraints are managed similarly to Constraint through Kubermatic CRD wrappers around the Gatekeeper Constraints, the difference being that Default Constraints are managed on the admin level by Kubermatic platform admins.
```

Comment:
maybe you can mention that Default Constraints reside in the `kubermatic` namespace, from where they are propagated to cluster namespaces